src/components/DocsHeader.tsx

```diff
@@ -24,6 +26,9 @@ export function DocsHeader({
 
   return (
     <header className="space-y-1">
+      <title>
+        {documentTitle ? `Replay docs: ${documentTitle}` : 'Replay docs'}
```

Comment:
Do we want to include the "Replay docs: " prefix? I'd kind of prefer to drop it but I wanted to check.